packages/getEntropy/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import { DialogType } from '@metamask/rpc-methods';
+import { ethErrors } from 'eth-rpc-errors';
+import { OnRpcRequestHandler } from '@metamask/snaps-types';
+import { bytesToHex, remove0x } from '@metamask/utils';
+import { sign } from '@noble/bls12-381';
+import { panel, text, heading } from '@metamask/snaps-ui';
+
+const getEntropy = async () => {
+  const entropy = await snap.request({
+    method: 'snap_getEntropy',
+    params: {
+      version: 1,
+    },
+  });
+  return remove0x(entropy);
+};
+
+export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
+  switch (request.method) {
+    case 'getEntropyOutput': {
```

Comment:
Since this is not used anymore, can you remove this?

Reply:
Of course! fixed in latest commit